common/ustr.cpp

```diff
@@ -428,7 +428,13 @@ void U32String::initWithCStr(const char *str, uint32 len) {
 	_str[len] = 0;
 }
 
-// This is a quick and dirty converter.
+// //TODO: This is a quick and dirty converter. Refactoring needed:
+// 1. This version is unsafe! There are no checks for end of buffer
+//    near i++ operations.
+// 2. Original version have an option for performing strict / nonstrict
```

Comment:
Nitpicking: “Original version has an option...”

Reply:
Thanks for pointing those. I'm not that good with English. Fixed at 63111560216a00afe31a7cee71062f22eda89a43